### PR TITLE
ENH: wrap of the warp code in Fiji

### PR DESCRIPTION
**Pull request by poilvert**
Add wrapping of the Fiji `warp` code. all tests pass.

### CODE REVIEW

File: bangmetric/isbi12/tests/test_isbi12.py [hunk 1]
```diff
@@ -50,3 +53,18 @@ def test_simple_segmentation_with_provided_path():
     assert np.abs(pe - 0.010183) < EPSILON
     assert np.abs(re - 0.020385) < EPSILON
     assert np.abs(we - 0.0) < EPSILON
+
+
+def test_warp_2d():
+
+    # -- read in y_pred and y_true from PNG images
+    y_pred = misc.imread('./y_pred.png', flatten=True)
```
Comment (npinto) [inline bangmetric/isbi12/tests/test_isbi12.py:61]:
```
E           IOError: [Errno 2] No such file or directory: './y_pred.png'
```

I'll fix it in master.

File: bangmetric/isbi12/tests/test_isbi12.py [hunk 2]
```diff
@@ -50,3 +53,18 @@ def test_simple_segmentation_with_provided_path():
...
+    y_true = misc.imread('./y_true.png', flatten=True)
+
+    # -- compute the "warped annotations"
+    y_warp = warp_2d(y_true, y_pred)
```
Comment (npinto) [inline bangmetric/isbi12/tests/test_isbi12.py:65]: argn, the test takes wayyy too long with 512x512...

**Comment by npinto** [comment]
There are still a bunch of stuff to fix, I'll merge the fixes in master. Closing this PR.

File: bangmetric/isbi12/tests/test_isbi12.py [hunk 3]
```diff
@@ -50,3 +53,18 @@ def test_simple_segmentation_with_provided_path():
...
+
+    matshow(y_pred)
+    matshow(y_true)
+    matshow(y_warp)
+    show()
```
Comment (npinto) [inline bangmetric/isbi12/tests/test_isbi12.py:70]: what is this? 
this is not a test...

Comment (poilvert) [inline bangmetric/isbi12/tests/test_isbi12.py:70]: I know it is not a test, but I don't know what metrics would be
used for that kind of operation aside visual inspection...

On Thu, Jul 26, 2012 at 4:33 PM, Nicolas Pinto <
reply@reply.github.com

**Pull request closed without merging**